docs/environment.yml

```diff
@@ -0,0 +1,21 @@
+name: jupyterlab-plugin-playground-lite
+
+channels:
+- conda-forge
+
+dependencies:
+- build
+- python=3.10
+- mamba
+- pydata-sphinx-theme
+- myst-parser
+- empack
```

Comment:
the overhead of this and the xeus stack doesn't seem to make a ton of sense on this particular lite site

Reply:
Indeed, this was copied from another repo. Makes sense to stay with the default kernel.